packages/analytics/package.json

```diff
@@ -7,6 +7,7 @@
         "typescript": "^4.6.3"
     },
     "dependencies": {
+        "@deriv/shared": "^1.0.0",
```

Comment:
@akmal-deriv could you please also run `npm run bootstrap:dev` to generate package-lock file based on your changes in package.json files? :)

Reply:
There's no package-lock in analytics package. I obviously ran it before)